tests/containers/docker.pm

```diff
@@ -47,8 +47,11 @@ sub run {
 
     if ($self->firewall() eq 'firewalld') {
         # on publiccloud we need to install firewalld first
-        install_and_start_firewalld() if (is_publiccloud || is_jeos);
-        check_docker_firewall();
+        zypper_call('install firewalld') if (is_publiccloud || is_jeos);
+        systemctl('restart firewalld');
```

Comment:
you might want to check `lib/services/firewall.pm`

Reply:
I thought the same, but after a quick look at that library, it's really missing a lot of things... To start with, `install_service` installs `SuSEfirewall2` instead of `firewalld`..., then, there is no `restart` method... 

However, what I would do is replace `firewalld` by `opensusebasetest::firewall`